addons/hr_skills/__manifest__.py

```diff
@@ -44,6 +44,7 @@
             'hr_skills/static/src/scss/*.scss',
             'hr_skills/static/src/views/*.js',
             'hr_skills/static/src/xml/**/*',
+            'hr_skills/static/src/**/*',
```

Comment:
> Should we just add the components sub-directory ? or remove all the rest and just add this ? because as I understand, your addition should include all sub directories of src.

I hesitated to do so but I was wondering if the order in which it was imported might have some importance. Anyway, I'll try to remove those lines and I suppose runbot will not be happy if it had an importance.

Reply:
Here is the answer I guess :sweat_smile: 
https://runbot.odoo.com/runbot/batch/1199405/build/52470683